docs/examples/laravel/buildpack-cloud-run/build_container_image_live.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+PHPBUILDER=$(tail -n 1 /workspace/phpbuilder)
+
+bash ./generate_env_live.sh
+
+pack build --builder eu.gcr.io/gae-runtimes/buildpacks/php74/builder:$PHPBUILDER --publish eu.gcr.io/$PROJECT_ID/example-service:$COMMIT_SHA --env-file ./.env
```

Comment:
Cool to know `pack` support the `--env-file` option!

I think the ".env" file is for use with Laravel, so it is deployed alongside the application and read from disk by the app on startup: we perhaps don't need to pass it in here?

The environment variable we do need is "GOOGLE_ENTRYPOINT" to setup logging.

Do we swap `--env-file ./.env` for `--env=GOOGLE_ENTRYPOINT=/workspace/vendor/a1comms/gae-support-laravel/gcr/entrypoint.sh`?

Or would you keep the env-file being passed to pack and add `GOOGLE_ENTRYPOINT` into there?

Reply:
Had noticed the .env for laravel thing myself, I suppose I was unsure if it would need any of it during build, but looking through the values it appears not, so happy to make that change.

Reply:
Will make the corresponding change within the [private repo this is also being utilised in] too.